lib/SPIRV/VectorComputeUtil.cpp

```diff
@@ -120,6 +120,10 @@ getVCGlobalVarStorageClass(SPIRAddressSpace AddressSpace) noexcept {
     return StorageClassPrivate;
   case SPIRAS_Local:
     return StorageClassWorkgroup;
+  case SPIRAS_Global:
+    return StorageClassCrossWorkgroup;
+  case SPIRAS_Constant:
+    return StorageClassUniformConstant;
```

Comment:
Note: this will most likely require `Shader` capability to be declared in SPIR-V module

Reply:
Hmm... I hope it is not. At least in the spec, the lists of enabling capabilities for StorageClassUniformConstant and StorageClassCrossWorkgroup are empty...

Reply:
Oh, my guess was incorrect then, sorry for confusion